src/test/java/com/hazelcast/kubernetes/KubernetesClientTest.java

```diff
@@ -0,0 +1,320 @@
+                        "    \"labels\": {" +
+                        "      \"app\": \"hazelcast\"," +
+                        "      \"chart\": \"hazelcast-1.0.0\"," +
+                        "      \"heritage\": \"Tiller\"," +
+                        "      \"release\": \"my-hazelcast\"" +
+                        "    }" +
+                        "  }," +
+                        "  \"subsets\": [" +
+                        "    {" +
+                        "      \"addresses\": [" +
+                        "        {" +
+                        "          \"ip\": \"%s\"," +
+                        "          \"nodeName\": \"minikube\"," +
+                        "          \"hazelcast-service-port\" : %s," +
+                        "          \"targetRef\": {" +
+                        "            \"kind\": \"Pod\"," +
+                        "            \"namespace\": \"default\"," +
+                        "            \"name\": \"my-hazelcast-0\"," +
+                        "            \"uid\": \"def2f426-8443-11e8-abd2-00155d395157\"," +
+                        "            \"resourceVersion\": \"18757\"" +
+                        "          }" +
+                        "        }," +
+                        "        {" +
+                        "          \"ip\": \"%s\"," +
+                        "          \"nodeName\": \"minikube\"," +
+                        "          \"hazelcast-service-port\" : %s," +
+                        "          \"targetRef\": {" +
+                        "            \"kind\": \"Pod\"," +
+                        "            \"namespace\": \"default\"," +
+                        "            \"name\": \"my-hazelcast-1\"," +
+                        "            \"uid\": \"f3b96106-8443-11e8-abd2-00155d395157\"," +
+                        "            \"resourceVersion\": \"18815\"" +
+                        "          }" +
+                        "        }" +
+                        "      ]," +
+                        "      \"ports\": [" +
+                        "        {" +
+                        "          \"name\": \"hzport\"," +
+                        "          \"port\": 5701," +
+                        "          \"protocol\": \"TCP\"" +
+                        "        }" +
+                        "      ]" +
+                        "    }" +
+                        "  ]" +
+                        "}"
+                , SAMPLE_ADDRESS_1, SAMPLE_PORT_1, SAMPLE_ADDRESS_2, SAMPLE_PORT_2);
+    }
+
+    /**
+     * Real response recorded from the Kubernetes API.
+     */
+    private static String forbiddenBody() {
+        return "Forbidden!Configured service account doesn't have access. Service account may have been revoked. "
+                + "endpoints is forbidden: User \"system:serviceaccount:default:default\" cannot list endpoints "
+                + "in the namespace \"default\"";
+    }
```

Comment:
yep we know this one 👍

Reply:
hehe, yeah, that's why I added a test specifically for this case ;)